clang-format.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
```

Comment:
I think I'm missing something. But, why do we need `check-clang.sh` when we have this script? If `check-clang.sh` is meant to be for checking `clang-format` existence, would it be better to put the script here?

Reply:
I am thinking the same. I am going to see which works out better. So testing it out.